keymanapp/keyman
# feat(windows): Add two new strings for SIL Global name instead of SIL International

Fixes: #12251

This updated the strings from SIL International to SIL Global. It also changed the format of the replacement string id to allow for translation of the surrounding text while keeping SIL Global untranslated. 
# User Testing
## TEST_SIL_GLOBAL
1. Install Keyman attached to PR
2. Open Configuration and go to the Support tab. Verify the text now says `SIL Global`
![image](https://github.com/user-attachments/assets/126c7382-818e-4908-9cee-970b58706d2a)
